modules/services/picom.nix

```
@@ -269,10 +271,25 @@ in {
       '';
     };
 
+    extraWintypes = mkOption {
+      type = types.str;
+      default = "";
+      example = literalExample ''
+        tooltip = {
+          fade = true;
+          shadow = true;
+        };
+        dock = { opacity = 0.75; };
+      '';
```

The example value does not match the types allowed by the option.

type should be `types.lines` and the example value should be a string and not an attribute set.

```suggestion
      type = types.lines;
      default = "";
      example = literalExample '''''
        tooltip = {
          fade = true;
          shadow = true;
        };
        dock = { opacity = 0.75; };
      ''''';
```